### PR TITLE
Add 1 Icon

### PR DESCRIPTION
**Pull request by Hamster45105**
# Description

### Added
~~NYT Games (`com.nytimes.crossword`)~~ (Added by someone else after PR was created)
Cricket Australia Live (`au.com.cricket`)

## Contributor's checklist

- [x] I followed [the Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) (upd. Jan 2024) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.

### CODE REVIEW

File: svgs/cricket_australia_live.svg
Comment (x9136) [inline svgs/cricket_australia_live.svg:1]: The distance between the shield and the text is too small. I also suggest improving the kerning.
https://99designs.com/blog/tips/11-kerning-tips/

File: svgs/nyt_games.svg
Comment (x9136) [inline svgs/nyt_games.svg:1]: There is not enough height to make a square.

**Review by x9136 — CHANGES_REQUESTED** [review]

File: svgs/cricket_australia_live.svg
Comment (Hamster45105) [reply svgs/cricket_australia_live.svg:1]: Thanks for the advice!

I've improved the kerning and the spacing of the shield and letters, but I'm not sure if the letters are too close together now?

Comment (x9136) [reply svgs/cricket_australia_live.svg:1]: > if the letters are too close together now?

That's for sure.

**Pull request merged**